sbol_utilities/sbol_diff.py

```diff
@@ -18,35 +19,32 @@ def _load_rdf(fpath: Union[str, bytes, os.PathLike]) -> rdflib.Graph:
     return graph1
 
 
-def _detect_sbol_version(graph: rdflib.Graph) -> Optional[str]:
+def _detect_sbol_version(graph: rdflib.Graph):
```

Comment:
```suggestion
def _detect_sbol_version(graph: rdflib.Graph) -> Optional[str]:
```
Why delete the type information?

Reply:
Outdated